libr/bin/bin.c

```diff
@@ -383,39 +386,38 @@ R_API bool r_bin_open_io(RBin *bin, RBinOptions *opt) {
 		// change the name to something like
 		// <xtr_name>:<bin_type_name>
 		r_list_foreach (bin->binxtrs, it, xtr) {
-			if (xtr->check_bytes (buf_bytes, opt->sz)) {
+			if (!xtr->check_buffer) {
```

Comment:
Many bin plugins do not have yet the check_buffer method :(

Reply:
this is just for binxtr plugins, and i created a wrapper method that takes the buffer and calls the buf+sz method internally for now, we have to kill all the bytes methods before 3.6

Reply:
Ah right, if it's just for xtr there are only few of them

Reply:
done